src/model/entity/item.go

```diff
@@ -0,0 +1,71 @@
+package entity
+
+import (
+	"math/rand"
+)
+
+const (
+	IncreaseAttributeBaseParcentage int = 5
+	IncreaseAttributeMaxPercentage  int = 20
+)
+
+type ItemLike interface {
+	Taken()
+	Dropped(box Box)
+	Use(p *Player)
+}
```

Comment:
Нужно разделить интерфейс на три:


```suggestion
type Takeable interface {
	Take()
}

type Droppable interface {
	Drop()
}

type Usable interface {
	Use(p *Player)
}
```